
# pre pick and post pick params [x, y, z] wrt robot base frame
pre_pick_offset: [-0.1, 0, 0]
post_pick_offset: [-0.43, 0, 0.3]

# pre-defined middle poses in joint states (rad)
## left
# pre_middle_left: [0.8, -1.7568, 2.5506, 0.7786, -1.5734, -0.0038]
pre_middle_left: [0.8004, -1.8953, 2.1806, 1.2872, -1.5733, -0.0037]
# post_middle_left: [1.3014, 0.7062, 1.9060, -0.0747, -0.0039, -0.0804]
post_middle_left: [1.3, 0.7062, 1.9060, -0.3747, -0.0039, -0.0804]

## right
# pre_middle_right: [0.8, 1.7568, -2.5506, -0.7786, 1.5734, 0.0038]
pre_middle_right: [0.8004, 1.8953, -2.1806, -1.2872, 1.5733, 0.0037]
# post_middle_right: [1.3014, -0.7062, -1.9060, 0.0747, 0.0039, 0.0804]
post_middle_right: [1.3, -0.7062, -1.9060, 0.3747, 0.0039, 0.0804]

# pre-defined place poses in joint states (rad)
## left
# place_left: [0.9706, 1.4689, 1.9270, -0.2326, 0.0, 0.0]
place_left: [1.2706, 1.8999, 2.2317, -1.5336, 0.0, 0.5644]

## left
# place_right: [0.9706, -1.4689, -1.9270, 0.2326, 0.0, 0.0]
place_right: [1.2706, -1.8999, -2.2317, 1.5336, 0.0, -0.5644]
# Parallel running parameters
# detected_obj_pose_position: [0,0,0]
# detected_obj_pose_orientation: [0,0,0,0]